packages/widget-v2/src/components/VirtualList.tsx

```diff
@@ -27,22 +27,25 @@ export const VirtualList = <T,>({
   const listRef = useRef<ListRef>(null);
 
   useEffect(() => {
-    const listElement = listRef.current?.nativeElement;
-    //@ts-expect-error ignore typescript error
-    const onFocus = (e) => {
-      setCurrentlyFocusedElement(e?.target);
-    };
-    listElement?.addEventListener("focusin", onFocus);
+    const listElement = listRef.current?.nativeElement?.getElementsByClassName("rc-virtual-list-holder-inner")?.[0];
+    const firstElementInList = (listElement?.firstChild as HTMLElement);
 
     const handleKeyDown = (event: KeyboardEvent) => {
+
       if (event.key === "ArrowDown") {
+        if (firstElementInList) {
+          firstElementInList.focus();
+          setCurrentlyFocusedElement(firstElementInList);
+        }
+
```

Comment:
@codingki yeah this makes it select the first item in the list when pressing the down arrow